packages/TorneloScoresheet/src/types/ChessGameInfo.ts

```diff
@@ -15,7 +15,7 @@ export type ChessGameInfo = {
 export type ChessGameResult = {
   winner: PlayerColour | null;
   gamePgn?: string;
-  signature: string[];
+  signature: { 0: string | undefined; 1: string | undefined };
```

Comment:
Why not Record<PlayerColor, string> ?
i think at this point the signature can never be undefined right?
Because there is no option to skip the signing step,

Reply:
If I do that I get this error 
![image](https://user-images.githubusercontent.com/64511606/182262401-f7ae602a-0549-49a3-a85b-c7e11f67832f.png)

Reply:
Yep because in the graphical page the signature can still be undefined, so we need to build a new one where we have a default if the signature is undefined (it should never be at that point though because were already checking on line 203
![image](https://user-images.githubusercontent.com/26475724/182264653-49ff54a2-ef1d-4423-8888-caa0898b6b58.png)